codecept.multiple.conf.js

```diff
@@ -0,0 +1,18 @@
+exports.config = {
+  tests: './test/acceptance/*_test.js',
+  output: './test/acceptance/output',
+  multiple: {
```

Comment:
No helpers? https://github.com/verdaccio/ui/blob/master/codecept.conf.js#L4

Reply:
https://codecept.io/helpers
https://github.com/Codeception/CodeceptJS/blob/master/test/acceptance/codecept.WebDriverIO.js#L8

Reply:
🙃 oops .... now it works ... thanks :)

Reply:
https://github.com/verdaccio/ui/pull/111/files#diff-c79d74b30446eba929960ff164233bf4R5